### PR TITLE
readme: Update readme to expand on usage

### PR DESCRIPTION
**Pull request by Volker-E**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Wikimedia Design Style Guide (WikimediaUI)
 
-The Wikimedia Design Style Guide ensures a consistent look and behavior for our products.
+The Wikimedia Design Style Guide ensures a consistent look and behavior for Wikimedia Foundation products.
```
Comment (Ladsgroup) [inline README.md:3]: hmm, should we have something like "WMDE" too? I don't know how to properly phrase it though.

Comment (Ladsgroup) [reply README.md:3]: What I mean is that this DSG is more broad than WMF products.

Comment (Volker-E) [reply README.md:3]: Yeah, I was actually aiming to write Movement… See Wikidata further down.

Comment (Ladsgroup) [reply README.md:3]: Ack.

**Review by Ladsgroup — APPROVED** [review]

**Review by Ladsgroup — APPROVED** [review]

**Pull request merged**